### DISCUSSION TITLE
Has "time-local" variable implementation changed?

### DISCUSSION THREAD

**Discussion by radiant-tangent** [gildas-lormeau/SingleFile#1631] [Q&A] [answered]
I have my filename format defined as follows: "%if-empty<{page-title}|No title> ({date-iso} {time-locale}).html"
I use singlefile on mac and windows with mainly edge and firefox.
On 11/19 time-locale when saved from firefox was "10_46_10 PM"
Since 11/24 time-local varies based on the browser I'm using.
For example, firefox now saves time-locale as "10: 46: 10 PM"
and edge (on mac) saves time-local as "10 | 46 | 10 PM"

Did the implementation of time-locale change and is there way to change my filename format so that time is formatted the same across operating systems and browsers?

**Comment by gildas-lormeau** [comment, marked as answer]
The behavior has indeed changed, see #1614.

To revert the changes, update the the hidden option `filenameReplacementCharacters` (see https://github.com/gildas-lormeau/SingleFile/wiki/Hidden-options) by replacing:

`"filenameReplacementCharacters": ["～", "＋", "？", "％", "＊", "：", "｜", "＂", "＜", "＞", "＼"],`

with:

`"filenameReplacementCharacters": ["_", "_", "_", "_", "_", "_", "_", "_", "_", "_", "_"],`

**Reply by radiant-tangent** [reply]
Making this change worked on edge but firefox is still giving ":" instead of "_". I have tried restarting the browser but still no change. I exported the settings again to verify they are all underscores (and its the file I imported into edge).

**Reply by gildas-lormeau** [reply]
The extension is not up to date on the Mozilla store as I write this reply. It will be available once the code has been reviewed by Mozilla.
